examples/cim/gnn/action_shaper.py

```diff
@@ -0,0 +1,42 @@
+import math
```

Comment:
Please follow the [import rules](https://github.com/microsoft/maro/wiki/Code-Style-Guide). Check and fix other files.